lib/boxr/files.rb

```diff
@@ -190,6 +190,13 @@ def copy_file(file, parent, name: nil)
       new_file, res = post(uri, attributes)
       new_file
     end
+      
+    def representations(file_id,representation)
+      file_id = ensure_id(file_id)
+      uri = "#{FILES_URI}/#{file_id}?fields=representations"
+      file, response = get(uri, x_rep_hints: "#{representation}")
```

Comment:
Is it necessary to interpolate the string here? I don't see anything being added to it

Reply:
To be honest it's been a while since I wrote this and don't remember exactly why I implemented this this way. With that in mind which string value are you referencing? 

Also, this method doesn't return the actual representation file, it only returns Representation Info
I've seen other SDK's that return the actual file, should we do the same?

https://github.com/box/box-node-sdk/blob/1d51f676b1323135891a70d470e2c9de97be9437/docs/files.md#get-representation-content

Reply:
I meant that 
```
x_rep_hints: "#{representation}"
``` 
could just be
```
x_rep_hints: representation
```

Reply:
As for the representation content, I agree we should add it to the library. It should be a separate action where you have to specify the representation type, and we can use the identifiers Box has for these representations.

As for the method you've already implemented, I think it's still good since there's a separate endpoint to fetch a list of the representations. 

If we want to fill out the feature, it seems we'll also want a method to hit the `info.url` in order to manually generate representations that were not automatically generated.

I'm fine breaking down this feature into multiple smaller parts, however.

Reply:
yeah... good point. I don't know why I added the additional string. I'll fix the code and also add tests.
We should probable breakdown the method to get the info.url endpoint.